client/src/features/form/FormMoverControl.js

```diff
@@ -55,6 +55,11 @@ const FormMoverControl = () => {
 
         checkForEmptyField("title");
         checkForEmptyField("description");
+        if (formData.description?.length > 280) {
+          errorArray.push(
+            "Error: Description must be less than 280 characters."
```

Comment:
I think this would add some context to the error:
```suggestion
            `Error: Description must be less than 280 characters. Current character count: ${formData.description?.length}.`
```